src/tigerbeetle-java/src/main/java/com/tigerbeetle/Client.java

```diff
@@ -26,43 +20,103 @@ public final class Client implements AutoCloseable {
     private long packetsHead;
     private long packetsTail;
 
-    public Client(int clusterID, String[] replicaAddresses) {
-        this(clusterID, replicaAddresses, DEFAULT_MAX_CONCURRENCY);
-    }
-
+    /**
+     * Initializes an instance of TigerBeetle client. This class is thread-safe and for optimal
+     * performance, a single instance should be shared between multiple concurrent tasks.
+     * <p>
+     * Multiple clients can be instantiated in case of connecting to more than one TigerBeetle
+     * cluster.
+     *
+     * @param clusterID
+     * @param replicaAddresses
+     * @param maxConcurrency
+     *
+     * @throws InitializationException If TigerBeetle an error occurred initializing this client.
```

Comment:
"If TigerBeetle an error" doesn't make sense. Is "TigerBeetle" supposed to be in this sentence?

Reply:
Copy and paste crap, sorry ;-)